tools/compare_build_systems.py

```diff
@@ -87,14 +95,17 @@
     "PICO_DEFAULT_PIOASM_OUTPUT_FORMAT",
     # Bazel always has picotool.
     "PICO_NO_PICOTOOL",
-    # TODO: Eventualy support.
-    "PICO_NO_COPRO_DIS",
-    "PICO_DEFAULT_RP2350_PLATFORM",
-    "PICO_GCC_TRIPLE",
-    "PICO_NO_FLASH",
-    "PICO_COPY_TO_RAM",
-    "PICO_RP2350_ARM_S_CONFIG_HEADER_FILES",
-    "PICO_RP2350_RISCV_CONFIG_HEADER_FILES",
+    # These aren't supported as build flags in Bazel. Prefer to
+    # set these in board header files like other SDK defines.
+    "CYW43_DEFAULT_PIN_WL_REG_ON",
+    "CYW43_DEFAULT_PIN_WL_DATA_OUT",
+    "CYW43_DEFAULT_PIN_WL_DATA_IN",
+    "CYW43_DEFAULT_PIN_WL_HOST_WAKE",
+    "CYW43_DEFAULT_PIN_WL_CLOCK",
+    "CYW43_DEFAULT_PIN_WL_CS",
+    "CYW43_PIO_CLOCK_DIV_INT",
+    "CYW43_PIO_CLOCK_DIV_FRAC",
```

Comment:
Does this need to be changed from `CYW43_PIO_CLOCK_DIV_FRAC` to `CYW43_PIO_CLOCK_DIV_FRAC8` to match the change in #1926 ?
Also, this list might be missing `CYW43_PIN_WL_DYNAMIC` which is one of the things that #2033 fixes in the CMake-world?

Reply:
Bleh, I forgot to pull latest changes before adding these. This just squelches the warnings, he findings shouldn't get in the way of anything. I can put up a separate PR to add those.